extension/app/scripts.babel/background.js

```diff
@@ -17,12 +17,17 @@
 
 'use strict';
 
-import {runPwaAudits} from './pwa-check.js';
+const ExtensionProtocol = require('../../../helpers/extension/driver.js');
+const runner = require('../../../runner');
+const driver = new ExtensionProtocol();
 
-window.runAudits = function() {
-  return runPwaAudits(chrome).then(results => {
-    return createResultsHTML(results);
-  }).catch(returnError);
+window.RELOAD_PAGE_AND_RUN_ALL_TESTS = true;
+
+window.runAudits = function(reloadPageAndRunAllTests) {
```

Comment:
I would much prefer an object with a descriptive name being passed into here, vs a flat boolean.

Read this:
http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html